presto-hive/src/main/java/io/prestosql/plugin/hive/authentication/HiveIdentity.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.authentication;
+
+import io.prestosql.spi.connector.ConnectorSession;
+import io.prestosql.spi.security.ConnectorIdentity;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public final class HiveIdentity
+{
+    private final String username;
+
+    private HiveIdentity()
+    {
+        this(new ConnectorIdentity("dummy_identity", Optional.empty(), Optional.empty()));
```

Comment:
Simply:

```suggestion
        this.username = "dummy_identity";
```

Reply:
Still, I'd prefer the `username` be `Optional<String>` in this class.
There is only one place where `getUsername()` is called, so it's not a big deal and improves clarify.